Dockerfile

```diff
@@ -1,30 +1,14 @@
-# Docker multi-stage build
+# switched to debian image to be able to use plantuml, graphviz and dot
+FROM debian:stable
 
-# 1. Building the App with Maven
-FROM maven:3-jdk-11
+RUN apt-get update && apt-get install -y plantuml graphviz && rm -rf /var/cache/apt/archives/*
+RUN apt-get update && apt-get install -y maven && rm -rf /var/cache/apt/archives/*
 
-ADD . /openapi2puml
-WORKDIR /openapi2puml
+COPY . /app
+WORKDIR /app
+RUN mvn clean install -DskipTests=true -Dgpg.skip -Dmaven.javadoc.skip=true -B -V
+RUN echo -e '#! /bin/bash\njava -jar /app/openapi2puml.jar $*' > /usr/local/bin/swagger2puml\
```

Comment:
Could you please explain what these two lines are doing? 
I wasn't sure so I tried removing them from the Dockerfile and I could still build and run the container without issues.
Also if they are needed could the swagger2puml be changed to openapi2puml?